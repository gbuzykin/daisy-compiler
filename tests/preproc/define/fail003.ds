#define MACRO(  

#define MACRO( ,

#define MACRO(x y

#define MACRO(__va_args__)