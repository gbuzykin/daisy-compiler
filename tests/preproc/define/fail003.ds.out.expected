./preproc/define/fail003.ds:1:17: error: expected macro argument identifier or `...`
 1 | #define MACRO(  
   |                 ^
./preproc/define/fail003.ds:3:16: error: expected macro argument identifier or `...`
 3 | #define MACRO( ,
   |                ^
./preproc/define/fail003.ds:5:17: error: expected `)` or `,`
 5 | #define MACRO(x y
   |                 ^
./preproc/define/fail003.ds:7:15: error: identifier `__va_args__` is reserved for variadic argument
 7 | #define MACRO(__va_args__)
   |               ^~~~~~~~~~~
./preproc/define/fail003.ds: info: warnings 0, errors 4
